assets/blocks/course-outline/course-block/edit.js

```diff
@@ -22,9 +18,10 @@ export const OutlineAttributesContext = createContext();
 /**
  * A hook to update the status store when a lesson is removed.
  *
- * @param {string} clientId The outline block id.
+ * @param {string}  clientId  The outline block id.
+ * @param {boolean} isPreview The outline block id.
```

Comment:
I think this description is not correct.

Reply:
Fixed in 4918b0ba63d82a80e7ca160bd1a37bcac204632a